libraries/AP_RangeFinder/AP_RangeFinder_LightWareI2C.cpp

```diff
@@ -84,7 +84,12 @@ bool AP_RangeFinder_LightWareI2C::get_reading(uint16_t &reading_cm)
     bool ret = _dev->read((uint8_t *) &val, sizeof(val));
     if (ret) {
         // combine results into distance
-        reading_cm = be16toh(val);
+        uint16_t distance = be16toh(val);
+        if (distance <= 10000) { /// max 100 meter
```

Comment:
this doesn't look right. some lightware lidars can do more than 100m

Reply:
@tridge san.  Thanks for your comment.
I will change the maximum value of the device, use the config parameter value.